includes/polling/ipmi.inc.php

```diff
@@ -35,6 +35,13 @@
             [$desc, $value, $type, $status] = explode(',', $row);
             $desc = trim($desc, ' ');
             $ipmi_unit_type = Config::get("ipmi_unit.$type");
+
+            // SDR records can include hexadecimal values, identified by an h
+            // suffix (like "93h" for 0x93). Convert them to decimal.
+            if (preg_match('/^[0-9A-Fa-f]+h$/', $value)) {
```

Comment:
Use this regex: `/^([0-9a-f]{2})+h$/i`

and $match[1] instead of substr to get the hex number on the next line.

Reply:
Updated. Though I removed the `{2}` because I don't have any documentation that assures me the values will always be 8 bits. If you're sure that's the case let me know and I'll add it back in. The only thing I know is that `h$` means hex, and thus that's the only assumption this change makes.

Reply:
Sure, I'm used to hex always being in chunks of 2 digits.  With leading 0s.